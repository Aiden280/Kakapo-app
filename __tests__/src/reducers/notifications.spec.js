import { compose, keys, values, head, length } from 'ramda';
import configureStore from 'stores/configureStore';
import { notifyActions } from 'actions/';

const store = configureStore();

const currState = () => store.getState().notifications;

test.skip('[reducer/notifications]', done => {
  expect.assertions(4);
  store.dispatch(notifyActions.send('test', 500));
  expect(compose(length, values)(currState())).toBe(1);
  expect(compose(head, values)(currState())).toBe('test');

  store.dispatch(notifyActions.send('testing', 500));
  setTimeout(() => {
    expect(compose(length, keys)(currState())).toBe(0);
    done();
  }, 1100);
});
