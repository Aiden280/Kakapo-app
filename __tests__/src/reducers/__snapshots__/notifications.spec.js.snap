// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[reducer/notifications] 1`] = `
Object {
  "2H200pPtv": undefined,
  "testID": "test",
}
`;

exports[`[reducer/notifications] 2`] = `
Object {
  "2H200pPtv": undefined,
}
`;
